noir-projects/aztec-nr/aztec/src/encrypted_logs/payload.nr

```diff
@@ -256,13 +269,13 @@ pub fn compute_outgoing_body_ciphertext(
     let serialized_recipient_address_point =
         point_to_bytes(recipient.to_address_point().to_point());
 
-    for i in 0..32 {
-        buffer[i] = serialized_eph_sk_high[i];
-        buffer[i + 32] = serialized_eph_sk_low[i];
-        buffer[i + 64] = address_bytes[i];
+    for i in 0..16 {
+        buffer[i] = serialized_eph_sk_high[i + 16];
+        buffer[i + 16] = serialized_eph_sk_low[i + 16];
```

Comment:
The first 16 bytes of `serialized_eph_sk_high/low` are 0s. We don't have to include them. This saves us 32 bytes.